website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**Major Enhancements:**
+* The HashiCorp Terraform MongoDB Atlas Provider now supports [Snapshot Distribution](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule) 
+  * To learn more about this feature see here: https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas   
+* The HashiCorp Terraform MongoDB Atlas Provider now allows users to [Programmatically Create API Keys](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key) 
+  * To learn more about this feature see here: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys
+  *  **IMPORTANT WARNING:**  Creating, Reading, Updating, or Deleting Atlas API Keys may expose sensitive organizational secrets to Terraform State. Consider storing sensitive API Key secrets instead via the [HashiCorp Vault MongoDB Atlas Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/mongodbatlas). 
+
+
+**Depreciations:**
+* [`cloud_provider_snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot), [`cloud_provider_snapshot_backup_policy`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_backup_policy), [`cloud_provider_snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job), and [`private_ip_mode`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/private_ip_mode) are now deprecated and will be removed from codebase as of v1.9
+* NEW_RELIC and FLOWDOCK in [`third_party_integration`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/third_party_integration#argument-reference) resource are now deprecated and will be removed from codebase as of v1.9 release 
```

Comment:
Please ensure the docs page for the resource and data sources mark these as deprecated: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/third_party_integration.  Assuming we also marked these as deprecated in code?  

Removed codebase (and from above) as that's not how users refer to it but how we, when developing, do. 
```suggestion
* NEW_RELIC and FLOWDOCK in [`mongodbatlas_third_party_integration`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/third_party_integration#argument-reference) resource are now deprecated and will be removed in the next major version, 1.9.
```

Reply:
updated. pulled in additional commits to reflect changes to third_party_integrations doc pages as well. also see here for deprecations in each of the respective code bases: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/988/files. also see here for details for ticket confirming third party integrations deprecations: https://jira.mongodb.org/browse/HELP-39527